modules/storage/storage-account/README.md

```diff
@@ -17,15 +17,46 @@ The output of the module is the ID of the created or existing Storage Account, w
+| `minimumTlsVersion`               | `string` | No       | Set the minimum TLS version to be permitted on requests to storage. The default interpretation is TLS 1.0 for this property.         |
+| `destPrefix`                      | `string` | No       | Prefix of destination Storage Account Resource Name. This param is ignored when name is provided.                                    |
+| `destStorageAccountName`          | `string` | No       | Name of destination Storage Account. Must be unique within Azure.                                                                    |
+| `destLocation`                    | `string` | Yes      | Deployment Location                                                                                                                  |
+| `destKind`                        | `string` | Yes      | Indicates the type of storage account.                                                                                               |
+| `destSupportHttpsTrafficOnly`     | `bool`   | No       | Allows https traffic only to storage service if sets to true.                                                                        |
+| `destAllowBlobPublicAccess`       | `bool`   | No       | Allow or disallow public access to all blobs or containers in the destination storage account.                                       |
+| `destAllowCrossTenantReplication` | `bool`   | No       | Replication of objects between AAD tenants is allowed or not. For this property, the default interpretation is true.                 |
+| `destPublicNetworkAccess`         | `string` | No       | Allow or disallow public network access to Storage Account. Value is optional but if passed in, must be Enabled or Disabled.         |
+| `destMinimumTlsVersion`           | `string` | No       | Set the minimum TLS version to be permitted on requests to storage. The default interpretation is TLS 1.0 for this property.         |
+| `destDaysAfterLastModification`   | `int`    | No       | It will be deleted after the given amount of days.                                                                                   |
+| `destBlobType`                    | `string` | No       | Specifies the type of blob to manage the lifecycle policy.                                                                           |
+| `destChangeFeedEnabled`           | `bool`   | No       | Indicates whether change feed event logging is enabled for the Blob service.                                                         |
+| `destVersioningEnabled`           | `bool`   | No       | Versioning is enabled if set to true. To the destination storage account, set true.                                                  |
+| `destSkuName`                     | `string` | Yes      | The SKU name required for account creation; optional for update.                                                                     |
```

Comment:
Can you expand on why this param is required? As well as how it can be optional for an update? Bicep deployments should not distinguish between create and update

Reply:
Assigned default value to this param and now it is optional.